charts/kube-prometheus-stack/README.md

```diff
@@ -80,6 +80,34 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 39.x to 40.x
+
+This version updates kube-state-metrics to 4.18.0 and prometheus-node-exporter to 4.1.0 which adds Kubernetes recommended labels on all resources.
+
+If you use your own custom [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitor) or [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#podmonitor), please ensure to upgrade their `selector` fields accordingly to the new labels.
+
+Starting from prometheus-node-exporter version 4.0.0, the `node exporter` chart is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therefore you have to delete the daemonset before you upgrade.
```

Comment:
please give exact instructions how to configure correct labels.

maybe we even need some other pr for nodeexporter first, adding something like: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml#L39-L40